src/sentry/interfaces/base.py

```diff
@@ -79,6 +79,7 @@ def path(cls):
     def external_type(cls):
         """The external name of the interface.  This is mostly the same as
         path with some small differences (message, debugmeta).
+        Also used as the display name for grouping strategy hints.
```

Comment:
Nit: There are presumably other places `external_type` is used, and we don't list them all, so not sure it's worth it to call out grouping info specifically.